### PR TITLE
[MO] - [system test] -> testing big amount of users

### PR DESCRIPTION
**Pull request by Seequick1**
### Type of change

- Enhancement / new feature

### Description

Performance test for checking if the users are creating correctly

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [x] Write tests
- [ ] Make sure all tests pass

### CODE REVIEW

**Comment by Seequick1** [comment]
[First Question]      -> I am not sure but it certainly won't damage the code if we create the same on SCRAM-SHA-512.
[Second Question] -> I just use the template from polarion, where manually was created 100 users. I can make some experiments with that numbers.

File: systemtest/src/test/java/io/strimzi/systemtest/UserST.java [hunk 1]
```diff
@@ -114,6 +117,26 @@ void testUpdateUser() {
         StUtils.waitForKafkaUserDeletion(kafkaUser);
     }
 
+    @Tag(PERFORMANCE)
+    @Test
+    void testBigAmountOfUsers(){
+        int numberOfUsers = 100;
+
+        for(int i = 0; i < numberOfUsers; i++){
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/UserST.java]: space after `for`

File: systemtest/src/test/java/io/strimzi/systemtest/UserST.java [hunk 2]
```diff
@@ -114,6 +117,26 @@ void testUpdateUser() {
...
+            String userName = "alisa" + i;
+            LOGGER.info("Creating user with name {}", userName);
+            testMethodResources().tlsUser(CLUSTER_NAME, userName).done();
+            StUtils.waitForSecretReady(userName);
+            LOGGER.info("Checking status of deployed Kafka User {}", userName);
+            Condition kafkaCondition = testMethodResources().kafkaUser().inNamespace(NAMESPACE).withName(userName).get()
+                    .getStatus().getConditions().get(0);
+            LOGGER.info("Kafka User Status: {}", kafkaCondition.getStatus());
+            LOGGER.info("Kafka User Type: {}", kafkaCondition.getType());
```
Comment (ppatierno) [inline systemtest/src/test/java/io/strimzi/systemtest/UserST.java]: Actually you are not logging the user type but the user status condition type.

File: systemtest/src/test/java/io/strimzi/systemtest/UserST.java (hunk 1 above)
Comment (Seequick1) [reply systemtest/src/test/java/io/strimzi/systemtest/UserST.java]: fixed

File: systemtest/src/test/java/io/strimzi/systemtest/UserST.java (hunk 2 above)
Comment (Seequick1) [reply systemtest/src/test/java/io/strimzi/systemtest/UserST.java]: fixed

**Comment by Frawless** [comment]
I am not sure if this test with multiple parameters will help us in that case. I think test it with one value will be fine and create some kind of marathon test, which will create and delete users in long time period (6-12h) together with msg exchange, topic creation, etc.

File: systemtest/src/test/java/io/strimzi/systemtest/UserST.java [hunk 3]
```diff
@@ -114,6 +117,26 @@ void testUpdateUser() {
         StUtils.waitForKafkaUserDeletion(kafkaUser);
     }
 
+    @Tag(PERFORMANCE)
```
Comment (Frawless) [inline systemtest/src/test/java/io/strimzi/systemtest/UserST.java]: Change it to scalability

**Pull request merged**